spec/wings/valkyrie/query_service_spec.rb

```diff
@@ -152,6 +152,24 @@ class Image < ActiveFedora::Base
       expect(found.id).to eq resource.id
       expect(found).to be_persisted
     end
+
+    context 'when use_valkyrie: false' do
+      it 'returns an ActiveFedora object' do
+        resource = resource_class.new
```

Comment:
Any reason not to set `id` outside the test using...
```
let(:id) do
  resource.alternate_ids = [Valkyrie::ID.new('p9s0xfj')]
  resource = persister.save(resource: resource)
  resource.alternate_ids.first
end
```

Reply:
I copied the code from an earlier spec and changed it for the "use_valkyrie" parameter. I'm not positive, but I believe that I split it out from the code elsewhere to placate Rubocop who thought line 163 was too long when it included `found = query_service.find_by_alternate_identifier(alternate_identifier: resource.alternate_ids.first, use_valkyrie: false)'. Naming it id is probably a bit unclear, given all of the other ids referenced in that spec.

Reply:
No problem.  This is more of a style thing.  I generally try to put setup code that isn't part of what is being tested outside the test itself.  Makes what is being tested easier to see at a glance.